src/Features/CSharp/Portable/Completion/KeywordRecommenders/NuintKeywordRecommender.cs

```diff
@@ -2,12 +2,20 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.CodeAnalysis.Completion;
 using Microsoft.CodeAnalysis.Completion.Providers;
 
 namespace Microsoft.CodeAnalysis.CSharp.Completion.KeywordRecommenders
 {
     internal sealed class NuintKeywordRecommender : AbstractNativeIntegerKeywordRecommender
     {
-        protected override RecommendedKeyword Keyword => new RecommendedKeyword("nuint");
+        /// <summary>
+        /// We set the <see cref="MatchPriority"/> of this item less than the default value so that completion selects
+        /// the <see langword="null"/> keyword over it as the user starts typing.  Being able to type <see
+        /// langword="null"/> with just <c>nu</c> is ingrained in muscle memory and is more important to maintain versus
+        /// strict adherence to our normal textual matching procedure.  The user can always still get this item simply
+        /// by typing one additional character and unambiguously referring to <c>nui</c>.
+        /// </summary>
+        protected override RecommendedKeyword Keyword => new RecommendedKeyword("nuint", matchPriority: MatchPriority.Default - 1);
```

Comment:
Maybe we should define a constant for this lower priority, instead of just `Default -1`

Reply:
```
const Default = 0;
const NotQuiteDefault = -1
```

:)

I thought about it, but didn't see a way that really helped out.